build.gradle

```diff
@@ -235,6 +235,38 @@ project(':iceberg-data') {
   }
 }
 
+project(':iceberg-flink') {
+  apply plugin: 'scala'
```

Comment:
Is it necessary to have Scala in the Iceberg build? We plan to remove it from the Spark build since it is only used for a couple of helpers.